phpunit.xml

```diff
@@ -3,33 +3,21 @@
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          bootstrap="test/phpunit-bootstrap.php"
          colors="true"
-         convertErrorsToExceptions="true"
```

Comment:
Think we might need to add the `--display-deprecations --fail-on-deprecation` and equivalent args for notice, warning, (and maybe risky?) to the phpunit command line to ensure it continues to be strict? From a quick look didn't see an alternative way to default that in the phpunit config file (which is 🙁)

Reply:
Oh I thought I'd read their new equivalents defaulted to on but it might have been part of an older discussion on a PR. Let me see if I can find it / understand it again

Reply:
I think now it collects them all and includes them as part of logging (unless you register a custom error handler), but doesn't by default show the details or exit nonzero to fail the build.

Feel like I ran into that somewhere (maybe a third-party package I contributed to) before...

Reply:
16525213e4b4f5220c275fc4d5489bb6a3aed05f might do the trick?